genfragments/ThirteenPointSixTeV/qqWWZZ_3l_TuneCP5_13p6TeV_powheg-pythia8.py

```diff
@@ -0,0 +1,41 @@
+import FWCore.ParameterSet.Config as cms
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.MCTunesRun3ECM13p6TeV.PythiaCP5Settings_cfi import *
+from Configuration.Generator.PSweightsPythia.PythiaPSweightsSettings_cfi import *
+from Configuration.Generator.Pythia8PowhegEmissionVetoSettings_cfi import *
+
+generator = cms.EDFilter(
+    "Pythia8ConcurrentHadronizerFilter",
+                         maxEventsToPrint = cms.untracked.int32(1),
+                         pythiaPylistVerbosity = cms.untracked.int32(1),
+                         filterEfficiency = cms.untracked.double(1.0),
+                         pythiaHepMCVerbosity = cms.untracked.bool(False),
+                         comEnergy = cms.double(13600.),
+                         PythiaParameters = cms.PSet(
+        pythia8CommonSettingsBlock,
+        pythia8CP5SettingsBlock,
+        pythia8PSweightsSettingsBlock,
+        pythia8PowhegEmissionVetoSettingsBlock,
+        processParameters = cms.vstring(
+          'POWHEG:nFinal = 4',   ## Number of final state particles
+              '23:mMin = 0.05',
+              '24:mMin = 0.05',
+              '23:onMode = off', # disable all Z decay modes
+              '23:onIfAny = 1 2 3 4 5 6 7 8 11 13 15', # enable only Z->ll and Z->qq
```

Comment:
Do you really need also 6, 7, and 8?

Reply:
Oh not really, we just included all by default, can be removed if necessary